src/test/kotlin/no/skatteetaten/aurora/gobo/graphql/credentials/CredentialMutationTest.kt

```diff
@@ -37,24 +43,30 @@ abstract class CredentialMutationTest : GraphQLTestWithoutDbhAndSkap() {
     val registerPostgresVariables = mapOf(
         "input" to jacksonObjectMapper().convertValue<Map<String, Any>>(postgresMotelInput)
     )
+
+    protected val messageSlot = slot<List<DetailedMessage>>()
+
+    @MockkBean
+    private lateinit var naghubService: NagHubService
+
+    @BeforeEach
+    fun setup() {
+        coEvery {
+            naghubService.sendMessage(any(), capture(messageSlot), any())
+        } returns NagHubResult(true)
+    }
 }
 
 @WithMockUser(
-    username = "system:serviceaccount:aup:vra"
+    authorities = ["testAdGroup"]
 )
-class AuthorizedAupTokenCredentialMutation : CredentialMutationTest() {
+class AuthorizedTokenCredentialMutation : CredentialMutationTest() {
```

Comment:
Alle disse testene, som ligger i denne filen og arver fra `CredentialMutationTest` går nettopp mot `CredentialMutation`. Hva med å lage en base klasse som heter `CredentialMutationTest` (rename den abstract klassen til noe annet), også putte de andre klassene inne i? Vi har gjort dette før og syns det blir ryddig når man trenger ulike klasser for ulike test cases.

Reply:
Det er nok ikke dumt! Skal se hva jeg får til! Det blir nok også da enklere å kjøre disse testene for intellij

Reply:
fikset